_includes/head.html

```diff
@@ -37,9 +37,6 @@
 
   <script src="https://cdn.jsdelivr.net/npm/vega-tooltip@0.5.1"></script>
   <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/vega-tooltip@0.5.1/build/vega-tooltip.min.css">
-
-  <script src="/assets/js/tag-filtering.js"></script>
```

Comment:
I was getting a console warning indicating this file couldn't be found and, indeed, I found no local file pertaining to this. Likewise, we get a warning about this JS file missing in production, so figured we ought to remove this for now!